projects/ngx-valdemort/src/test.ts

```diff
@@ -5,20 +5,5 @@ import 'zone.js/testing';
 import { getTestBed } from '@angular/core/testing';
 import { BrowserDynamicTestingModule, platformBrowserDynamicTesting } from '@angular/platform-browser-dynamic/testing';
 
-declare const require: {
-  context(
-    path: string,
-    deep?: boolean,
-    filter?: RegExp
-  ): {
-    keys(): Array<string>;
-    <T>(id: string): T;
-  };
-};
-
 // First, initialize the Angular testing environment.
 getTestBed().initTestEnvironment(BrowserDynamicTestingModule, platformBrowserDynamicTesting());
-// Then we find all the tests.
```

Comment:
you can add the errorOnUnknownElements config here or even remove the file if it doesn't load speculoos matches (and maybe the other one as well);

Reply:
In a later PR, I add the speculoos matchers